medusa/providers/torrent/json/t411.py

```diff
@@ -73,10 +89,90 @@ def __init__(self):
+                else:
+                    episode_string = (show_name, episode.scene_season, episode.scene_episode,)
+
+            search_string['Season'].append(episode_string)
+        return [search_string]
+
+    def _get_episode_search_strings(self, episode, add_string=''):
+        """Get episode search strings."""
+        if not episode:
+            return []
+
+        search_string = {
+            'Episode': []
+        }
+
+        for show_name in episode.show.get_all_possible_names(season=episode.scene_season):
+            episode_string = show_name + self.search_separator
+            if episode.show.air_by_date:
+                episode_string += str(episode.airdate).replace('-', ' ')
+                episode_string = episode_string.strip()
+            elif episode.show.sports:
+                episode_string += str(episode.airdate).replace('-', ' ')
+                episode_string += ('|', ' ')[len(self.proper_strings) > 1]
+                episode_string += episode.airdate.strftime('%b')
+                episode_string = episode_string.strip()
+            elif episode.show.anime:
+                # If the showname is a season scene exception, we want to use the indexer episode number.
+                if (episode.scene_season > 1 and
+                    show_name in get_scene_exceptions(episode.show.indexerid,
+                                                      episode.show.indexer,
+                                                      episode.scene_season)):
+                    # This is apparently a season exception, let's use the scene_episode instead of absolute
+                    ep = episode.scene_episode
+                else:
+                    ep = episode.scene_absolute_number
+                episode_string += '{episode:0>2}'.format(episode=ep)
+                episode_string = episode_string.strip()
+            else:
+                # Custom string for param search
+                if episode.scene_season > max(SEASON_MAP) or episode.scene_episode > max(EPISODE_MAP):
+                    episode_string += config.naming_ep_type[2] % {
+                        'seasonnumber': episode.scene_season,
+                        'episodenumber': episode.scene_episode,
+                    }
+                    episode_string = episode_string.strip()
+                else:
+                    episode_string = (show_name, episode.scene_season, episode.scene_episode,)
+
+            search_string['Episode'].append(episode_string)
+
+        return [search_string]
+
     def search(self, search_strings, age=0, ep_obj=None):
         """Search a provider and parse the results.
 
-        :param search_strings: A dict with mode (key) and the search value (value)
+        Provider has specific params for season and episode so:
+        - If is a normal episode/season search it will use URL params for season and episode.
+          In this case search string will be a list of tuples: [(show, season, episode)]
```

Comment:
It breaks the api; not as in does not work, but as in the api defines that search_strings is strings to search with.  By passing a tuple instead you are violating that contract.  The api needs to be extended to handle additional arguments instead.